### ISSUE TITLE
Creating event with GPS but no address name generates error

### ISSUE THREAD

**Issue by victor-priceputu** [spatie/icalendar-generator#19]
When trying to create an event with an address string but no name, and coordinates, the calendar generates error.

`Event::resolveLocationProperties()` sends nullable parameter `$this->addressName`, but the `Parameter::create()` method does not accept `NULL` as the second parameter.

**Comment by rubenvanassche** [comment]
Hi @victor-priceputu,

Could you provide an example of your code? Because when I create an event only with an address everything is working fine.

**Comment by victor-priceputu** [comment]
Sure thing, the code itself that I have is something like this:
```
$calendar = Calendar::create('Assessments Calendar')
            ->refreshInterval(5)

$event = Event::create('Custom Event')
            ->description('Random description')
            ->startsAt(Carbon::now())
            ->endsAt(Carbon::now()->addHour())
            ->address('Random Address, 'Venue location')
            ->coordinates(1, 1);

$calendar->event($event)
 ->get();
```

If I remove the `'Venue location'`, so the address does not have a name, I get an 500 error. If I remove the coordinates AND the address name, it all works. The event is in a loop so I add more than just one, but they are all the same in terms of what I add to it.

Then I return the data from the calendar in my response.

**Comment by rubenvanassche** [comment]
Hi @victor-priceputu this issue is fixed in release 1.0.3, will close this issue now!
